projects/examples/src/components/datagrid/datagrid-detail-row.example.component.ts

```diff
@@ -40,3 +47,25 @@ export class DatagridDetailRowExampleComponent {
         };
     }
 }
+
+@Component({
+    selector: 'vcd-datagrid-detail-row-sub-example',
+    template: `
+        <div [clrLoading]="loading">
+            {{ config.record }} : Record<br />
```

Comment:
` {{ config.record }}` will not show anything meaningful, maybe `JSON.stringify()`